digitalin.py

```diff
@@ -43,8 +43,8 @@ def load_xml(self, node: ET.Element):
         
         self.is_initialized = False
         
-        assert (node.tag == self.expectedRoot,
-                f"Expected tag <{self.expectedRoot}>, but received <{node.tag}>")
+        assert node.tag == self.expectedRoot, \
```

Comment:
you have to use a '\' line continuation here. rather than acting as a line continuation, your parentheses make the assert argument a tuple, and assert of a tuple will always return true